packages/client/src/forms/index.ts

```diff
@@ -97,7 +97,7 @@ export interface IFormConfig {
 
 export interface ISelectOption {
   value: SelectComponentOption['value']
-  label: MessageDescriptor
+  label: MessageDescriptor | string
```

Comment:
I don't think we need to change this to `MessageDescriptor | string`. We have two types defined in this file. One is using the `MessageDescriptor` and the other is basic `string`. Similarly in the `FormFieldGenerator` there are two stages, the first one gets the `MessageDescriptor`, converts it to `string` and forwards that to the second stage. So could you please try to figure out how to maintain this structure instead of changing it?